docs/galaxy/docsite/README.md

```diff
@@ -0,0 +1,48 @@
+Homepage and Documentation Source for Ansible Galaxy
+====================================================
+
+This project hosts the source behind [docs.ansible.com/galaxy](https://docs.ansible.com/galaxy)
+
+Contributions to the documentation are welcome. To make changes, submit a pull request that changes the reStructuredText files in the `rst/` directory only, and the core team can do a docs build and push the static files.
+
+If you wish to verify output from the markup such as link references, you may install sphinx and build the documentation by running `make webdocs` from the `ansible/docsgalaxy/docsite` directory.
+
+To include module documentation you'll need to run `make webdocs` at the top level of the repository. The generated html files are in `docsite/htmlout/`.
+
+To limit module documentation building to a specific module, run `MODULES=NAME make webdocs` instead. This should make testing module documentation syntax much faster. Instead of a single module, you can also specify a comma-separated list of modules. In order to skip building documentation for all modules, specify non-existing module name, for example `MODULES=none make webdocs`.
+
+If you do not want to learn the reStructuredText format, you can also [file issues] about documentation problems on the Ansible GitHub project.
+
+To install sphinx and the required theme, install ``pip`` and then ``pip install sphinx sphinx_rtd_theme``
+
+[file issues]: https://github.com/ansible/ansible/issues
```

Comment:
A link to https://github.com/ansible/ansible/issues/new?template=documentation_report.md would be quicker.